smac/utils/multi_objective.py

```diff
@@ -0,0 +1,30 @@
+from typing import Union, List, Tuple
+import numpy as np
+
+
+def normalize_costs(values: Union[np.ndarray, list],
+                    bounds: List[Tuple[float, float]] = None) -> np.ndarray:
```

Comment:
Some doc on `normalize_costs` requiring a 2 dimensional array, list would be nice :)

Also, just incase you're interested with respect to `list` or `List` for typing: https://stackoverflow.com/a/62033243

I've resigned to just using `List` everywhere for now and update it later once the deprecation warnings start to hit (years from now)